indica/readers/abstractreader.py

```diff
@@ -219,14 +219,20 @@ def get_ppts(
                 dims = ["t"]
             else:
                 raise ValueError(f"Unknown quantity: {quantity}")
+            if quantity == "R_shift":
```

Comment:
I don't understand where this is throwing an error. In assign_dataarray, if there is not ".._error" quantity, then a default error = xr.zeros_like(data) is assigned. 

I haven't had any issues with this reader and have included it in both the Zeff and the Prad workflow.

If possible, I would sort this out where you get the error and leave this reader method as it was.

Reply:
The issue is that the R_shift gets a default error and then that propogates to the equilibrium which then gives me some error when I try plasma.set_equilibrium

Reply:
What error? 
I'd rather find out what's going wrong, eventually changing the R_shift assignment in the Equilibrium if you needed.